go.mod

```diff
@@ -65,16 +66,14 @@ require (
 	github.com/mitchellh/go-homedir v1.1.0
 	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
 	github.com/modern-go/reflect2 v1.0.1 // indirect
-	github.com/nats-io/gnatsd v1.4.1 // indirect
-	github.com/nats-io/go-nats-streaming v0.4.4 // indirect
-	github.com/nats-io/nats-server v1.4.1 // indirect
-	github.com/nats-io/nats-streaming-server v0.14.2 // indirect
-	github.com/nats-io/nats.go v1.8.0
+	github.com/nats-io/nats-server/v2 v2.0.0 // indirect
```

Comment:
Please only update the necessary packages - in this PR it should be just `kurin/blazer` @mzukowski-reef

Reply:
@harshavardhana done